src/main/java/core/mvc/tobe/scanner/ControllerMethod.java

```diff
@@ -0,0 +1,52 @@
+package core.mvc.tobe.scanner;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import core.annotation.web.RequestMapping;
+import core.annotation.web.RequestMethod;
+import core.mvc.tobe.HandlerExecution;
+import core.mvc.tobe.HandlerKey;
+import core.mvc.tobe.support.ArgumentResolver;
+
+public class ControllerMethod {
```

Comment:
```suggestion
public class ControllerMethods {
```
일급 컬렉션 같은 느낌이 들어서 복수형으로 네이밍을 해봐도 좋을 것 같아요!